apps/charge-accounts-service/src/auth/guards/jwt-auth.guard.ts

```diff
@@ -1,5 +1,6 @@
 import { Injectable } from '@nestjs/common'
 import { AuthGuard } from '@nestjs/passport'
+import { simpleJwtString } from '@app/accounts-service/auth/auth.constants';
 
 @Injectable()
-export class JwtAuthGuard extends AuthGuard('jwt') {}
+export class JwtAuthGuard extends AuthGuard(['jwt', simpleJwtString]) {}
```

Comment:
Create a new JWT guard separately for operators instead of using this please

Reply:
@NikolayRn simple JWT will be extended to all instances where AuthGuard is used, such as in smart wallet service. Rather than adding the simple JWT guard individually into each endpoint where AuthGuard is already in place, it is more efficient to extend it within AuthGuard.

Reply:
Ok, i just checked all needed for your task endpoints works with both of JWT`s correctly, so yes, this solution is valid